src/site/content/en/blog/godaddy/index.md

```diff
@@ -0,0 +1,97 @@
+subhead: >
+  A case study of changes GoDaddy implemented to improve website performance for millions of sites, helping them to achieve good PageSpeed Insights and Core Web Vitals scores.
+description: >
+  A case study of changes GoDaddy implemented to improve website performance for millions of sites, helping them to achieve good PageSpeed Insights and Core Web Vitals scores.
+date: 2022-07-14
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/v5DPan0l6c9gOercxr3p.jpg
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/dUBhcdiLqE1OCDUV7ZYm.jpg
+alt: A time-lapse photo of a highway at night.
+tags:
+  - blog
+  - case-study
+  - web-vitals
+  - performance
+---
+
+GoDaddy is the world's largest services platform for entrepreneurs around the globe. We're on a mission to empower our worldwide community of 20+ million customers—and entrepreneurs everywhere—by giving them all the help and tools they need to grow online.
+
+In 2019 GoDaddy launched Websites + Marketing with a commitment to deliver tools and services that are easy to use and help business owners achieve their goals. Websites + Marketing integrates website building with marketing and eCommerce tools and pairs them with best-in-class guidance to help customers achieve success with their new ventures.
+
+Since the launch of Websites + Marketing, performance has been an important part of the product and something that has been monitored and worked on actively. In this article, we will review GoDaddy's journey from using lab performance testing to using real world data with Core Web Vitals, and the series of improvements made to the product to get a very high pass rate for our customer's sites.
+
+## Tracking performance with Lighthouse
+
+We have relied on Lighthouse data for performance tracking. Every time a website is published on the platform, we measure its performance using our internal tool named "Lighthouse4u", which provides [Google Lighthouse](https://github.com/GoogleChrome/lighthouse) as a service [https://github.com/godaddy/lighthouse4u](https://github.com/godaddy/lighthouse4u). This gave us a good indication of how sites were generally performing in a [lab setting](/lab-and-field-data-differences/#lab-data).
+
+Because the millions of sites we host on our platform have a wide range of features and content, it was important to combine performance data with metadata about each site being tested (template used, type of widgets rendered, etc.). This allowed us to draw conclusions on what website features had lower performance, while also providing insight for where to look for improvements. 
+
+For instance, this helped us identify that the "popup modal" was having a negative impact on page speed; sites with the feature performed 12 points lower than without. After making updates to the code to defer loading of JavaScript, we improved our Lighthouse score by 2 points. We were able to apply this learning to other features such as the "cookie banner" that renders soon after page load. 
```

Comment:
```suggestion
For example, this helped us identify that the "popup modal" was having a negative impact on page speed; sites with the feature performed 12 points lower than without. After making updates to the code to defer loading of JavaScript, we improved our Lighthouse score by 2 points. We were able to apply this learning to other features such as the "cookie banner" that renders soon after page load. 
```

Reply:
You can link to the cookie best practice article here https://web.dev/cookie-notice-best-practices/